Zend/zend_execute_API.c

```diff
@@ -851,6 +851,9 @@ int zend_call_function(zend_fcall_info *fci, zend_fcall_info_cache *fci_cache) /
 		if (EXPECTED((func->op_array.fn_flags & ZEND_ACC_GENERATOR) == 0)) {
 			zend_init_execute_data(call, &func->op_array, fci->retval);
 			zend_execute_ex(call);
+			if (UNEXPECTED(ZEND_CALL_INFO(call) & ZEND_CALL_CLOSURE)) {
+        			OBJ_RELEASE((zend_object*)func->op_array.prototype);
+			}
```

Comment:
I don't understand this and the following chunks. I think you were trying to fix memory leak in Zend/tests/closure_063.phpt, but I can't understand how these permutations do this.

The main problem here - Zend Engine just doesn't support INTERNAL closures. To properly support them we have to add similar OBJ_RELEASE code into DO_FCALL handler for INTERNAL functions.

BUT, this will slowdown every INTERNAL function call :(